src/openzaak/components/catalogi/api/admin/views.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2022 Dimpact
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework import authentication, permissions
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from openzaak.selectielijst.admin_fields import get_selectielijst_resultaat_choices
+
+from ...admin.forms import EMPTY_SELECTIELIJSTKLASSE_CHOICES
+from ...models import ZaakType
+
+
+class SelectielijstResultatenListView(APIView):
```

Comment:
Not sure what causes the issue, but it seems that the active values of `ZaakType.selectielijst_procestype_jaar` and `ZaakType.selectielijst_procestype` are not properly displayed, they always seem to show the defaults from the config instead of the actual values on the `ZaakType`